src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Android/Airship.cs

```diff
@@ -172,6 +173,49 @@ public Channel.TagGroupsEditor EditChannelTagGroups()
             });
         }
 
+        public AttributeEditor EditAttributes()
+        {
+            return new AttributeEditor((List<AttributeEditor.IAttributeOperation> operations) =>
+            {
+                var editor = UAirship.Shared().Channel.EditAttributes();
```

Comment:
UAChannel.Shared().something?

Reply:
this one is for android, so that looks right to me